plugins/community-plugin-xlsx/src/actions/DownloadXlsx.js

```diff
@@ -0,0 +1,33 @@
+import { type } from '@lowdefy/helpers';
+import writeXlsxFile from 'write-excel-file';
+
+async function DownloadXlsx({ params }) {
+  const { data, fileName, schema, ...options } = params;
+
+  if (!type.isArray(data) || !type.isObject(data[0])) {
+    throw new Error('Data should be an array of objects.');
+  }
+
+  const colTypes = {
+    String: String,
+    Number: Number,
+    Boolean: Boolean,
+    Date: Date,
+  };
+
+  await writeXlsxFile(data, {
+    fileName: !type.isString(fileName) ? 'download.xlsx' : fileName,
+    schema:
+      type.isArray(schema) &&
+      schema.map((column) => ({
+        ...column,
+        value: type.isString(column.value) ? (row) => row[column.value] : column.value,
+        type: column.type ? colTypes[column.type] : undefined,
```

Comment:
This change excludes the "formula" type. Change to:
```js
type: typs.isString(column.type) ? colTypes[column.type] ?? column.type : undefined,
```

Reply:
https://github.com/lowdefy/community-plugins/pull/24/commits/22f09a95cd2b6698db247ba4e1df6c44ec47c5ec